src/frontend/src/lib/components/auth/AuthGuard.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import LandingPage from '$lib/components/auth/LandingPage.svelte';
+	import { authNotSignedIn } from '$lib/derived/auth.derived';
+</script>
+
+{#if $authNotSignedIn}
+	<LandingPage />
+{:else}
+	<slot />
```

Comment:
Is the transition smooth enough? Or should we add a `div in:fade`?

Reply:
so, i tried with a few transitions but maybe it is too fast, or the first elements are too fast...

I am adding it, just to be smooth in any case!